presto-main/src/main/java/com/facebook/presto/operator/ExchangeClient.java

```diff
@@ -134,6 +138,10 @@ public ExchangeClient(
         this.systemMemoryContext = systemMemoryContext;
         this.maxBufferRetainedSizeInBytes = Long.MIN_VALUE;
         this.pageBufferClientCallbackExecutor = requireNonNull(pageBufferClientCallbackExecutor, "pageBufferClientCallbackExecutor is null");
+        this.averageBytesPerRequestEMA = new ExponentialMovingAverage(averageBytesEMAAlpha, DEFAULT_MAX_PAGE_SIZE_IN_BYTES);
+
+        checkArgument(averageBytesEMAAlpha >= 0.0, "averageBytesEMAALpha must be at least 0: %s", averageBytesEMAAlpha);
```

Comment:
How about merging two `checkArgument` into a single statement.

e.g.: `checkArgument(0 <= alpha && alpha <=1, "alpha must be between 0 and 1: %s", alpha)`

Reply:
Merged